### PR TITLE
work on routing_new example

### PR DESCRIPTION
**Pull request by mmoadeli**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/routing/308)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @inetic

(maidsafe_highfive has picked a reviewer for you, use r? to override)

**Comment by benjaminbollen** [comment]
please don't have the traces of this mornings debug; ie dont have the interface commented out

**Comment by mmoadeli** [comment]
Didn't notice the code had been commented out.

**Comment by benjaminbollen** [comment]
part of MAID-1066, but already merging for debug purposes; to be followed up

File: Cargo.toml [hunk 1]
```diff
@@ -28,3 +28,15 @@ name = "routing"
 path = "examples/routing.rs"
 test = true
 bench = false
+
+[[example]]
+name = "routing_new"
```
Comment (rossmuir) [inline Cargo.toml]: Can this have a more descriptive name please

File: Cargo.toml [hunk 2]
```diff
@@ -28,3 +28,15 @@ name = "routing"
...
+path = "examples/routing_new.rs"
+test = true
+bench = false
+
+[[example]]
+name = "routing_first"
```
Comment (rossmuir) [inline Cargo.toml]: Also this one - so it is obvious what is does - thanks

File: Cargo.toml (hunk 1 above)
Comment (mmoadeli) [inline Cargo.toml]: after Ben's changes are in place, routing_new will change to routing

File: Cargo.toml (hunk 2 above)
Comment (mmoadeli) [inline Cargo.toml]: I removed this one @rossmuir

**Comment by benjaminbollen** [comment]
thanks, I've included this work in PR https://github.com/maidsafe/routing/pull/310

**Pull request closed without merging**